### ISSUE TITLE
Indenting and consistant code style in spec

### ISSUE THREAD

**Issue by marcoscaceres** [sysapps/runtime#12]
I'm wondering if we can run the spec over [HTML5 tidy](https://github.com/w3c/tidy-html5) to give it consistant indenting, etc. Or is something else being used to indent the markup? This will make editing a lot easier. 

Right now, I need to write out new proposed text, move it to a new file, run tidy over the file, and copy the beautified code before sending a pull request. Having the original document alway be "tidy" would save (me, and possible other contributors) a bit of time.  

_cough_ _cough_ and moving to anolis would also be awesome _cough_ _cough_

**Comment by ming-jin** [comment]
Hi @marcoscaceres,

First of all, please fix a typo and tag mismatch error in index.html:

```
1374c1374
<             as the first argument is a Function object rather than a string;<il>
---
>             as the first argument is a Function object rather than a string;</li>
1377c1377
<             myElement.onXXX = someFunction;<li>
---
>             myElement.onXXX = someFunction;</li>
```

And then run html5 tidy with following tidy configuration:

```
indent: auto
indent-spaces: 2
wrap: 72
markup: yes
output-xml: no
input-xml: no
show-warnings: yes
numeric-entities: yes
quote-marks: no
quote-nbsp: yes
quote-ampersand: no
char-encoding: utf8
break-before-br: no
uppercase-tags: no
uppercase-attributes: no
new-inline-tags: cfif, cfelse, math, mroot, 
  mrow, mi, mn, mo, msqrt, mfrac, msubsup, munderover,
  munder, mover, mmultiscripts, msup, msub, mtext,
  mprescripts, mtable, mtr, mtd, mth, todo
new-blocklevel-tags: cfoutput, cfquery, section
new-empty-tags: cfelse
```

You will see that the whole runtime spec will get tidy-up properly :)

**Comment by marcoscaceres** [comment]
Excellent, thanks for the config file! I'll wait till the @mounirlamouri has decided which pull requests are going in before fixing the indenting. That will avoid some potential merge conflicts and make the pull requests easier to review.

**Comment by marcoscaceres** [comment]
So, I've gone through the [options for tidy](http://w3c.github.io/tidy-html5/quickref.html) and all we need in the configuration document is: 

```
char-encoding: utf8
indent: yes
wrap: 80
```

**Comment by marcoscaceres** [comment]
closed by #60
